docs/using-the-aws-driver/OktaAuthentication.md

```diff
@@ -0,0 +1,51 @@
+### Okta Authentication Plu# Okta Authentication Plugin
+
+The Okta Authentication Plugin adds support for authentication via Federated Identity and then database access via IAM.
+
+## What is Federated Identity
+
+Federated Identity allows users to use the same set of credentials to access multiple services or resources across
+different organizations. This works by having Identity Providers (IdP) that manage and authenticate user credentials,
+and Service Providers (SP) that are services or resources that can be internal, external, and/or belonging to various
+organizations. Multiple SPs can establish trust relationships with a single IdP.
+
+When a user wants access to a resource, it authenticates with the IdP. From this an authentication token generated and is
+passed to the SP then grants access to said resource.
+In the case of AD FS, the user signs into the AD FS sign in page. This generates a SAML Assertion which acts as a
+security token. The user then passes the SAML Assertion to the SP when requesting access to resources. The SP verifies
+the SAML Assertion and grants access to the user.
+
+## How to use Okta Authentication?
+
+> [!NOTE]\
+> AWS IAM database authentication is needed to use Okta Authentication. This is because after the plugin
+> acquires SAML assertion from the identity provider, the SAML Assertion is then used to acquire an AWS authentication token. The
+> AWS authentication token is then subsequently used to access the database.
+
+1. Enable AWS IAM database authentication on an existing database or create a new database with AWS IAM database authentication on the AWS RDS Console:
+    1. If needed, review the documentation about [creating a new database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html).
+    2. If needed, review the documentation about [modifying an existing database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Modifying.html).
```

Comment:
This is just a nitpick consistency thing. On the first point we went `modify` then `create`, but the bullets listed below go `create` then `modify`.